### PR TITLE
pop-up on status

### PR DESCRIPTION
**Pull request by sivxkxn**

### CODE REVIEW

File: src/app/shared/enum/applications.ts [hunk 1]
```diff
@@ -13,4 +13,13 @@ export enum ApplicationTitles {
   'Зараховано',
   'Відмовлено',
   'Гурток залишено',
+  'Заблоковано'
+}
+export enum ApplicationStatusDescription {
+  'Усі',
```
Comment (dmitryy90) [inline src/app/shared/enum/applications.ts:19]: lets discuss  how we can store all titles and descriptions to support multi languages

File: src/app/shell/personal-cabinet/personal-cabinet.component.html [hunk 2]
```diff
@@ -7,9 +7,35 @@ <h1 class="title">КАБІНЕТ КОРИСТУВАЧА</h1>
...
+    <a *ngIf="userRole === Role.parent" mat-tab-link [routerLinkActive]="['active']"
+      [routerLink]="'./parent/favorite'">УЛЮБЛЕНЕ</a>
+    <img src="assets/icons/application_info.png" alt="application-info" [matMenuTriggerFor]="menu"
+      #trigger="matMenuTrigger" (mouseenter)="trigger.openMenu()" (mouseleave)="trigger.closeMenu()">
+    <mat-menu [hasBackdrop]="false" disabled #menu="matMenu" [overlapTrigger]="false" (mouseleave)="closeMyMenu()"
+      class="mat-application-menu">
+      <div class="application-info-menu-title"> Статуси заявок</div>
+      <div class="application-info-menu">
+        <div class="application-info-title">
+          <span class="application-pending"><img src="assets/icons/Pending.png">Очікує підтвердження</span>
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/personal-cabinet.component.html:22]: why dont you use values from enum?

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Review by litvinets — CHANGES_REQUESTED** [review]

**Comment by litvinets** [comment]
1. I believe that info-box should appear when you click on the info icon. Because when you accidentally hover over it, the pop-up covers half of the page. Also, it should not disappear when you stop hovering the info-icon, because otherwise to read all the info the user should not move the mouse. 
2. It should be placed in the application tab but not inside the personal-cabinet tabs.
image
image
3.The info icon should be placed in the right upper corner for parent, and 
image and for provider in the same place but before the check-box-dropdown 
image

**Comment by litvinets** [comment]
1. don't use images for icons, always look for mat-icon!
2. You created an enum for application statuses, so use it in your component, otherwise, why do you need it?
3. I think we should create a separate component for this information and open it on hovering. Use it everywhere you need (both in the tab and in the application cards). For the card case, you can pass as input the status you need and display it in the info-box component. The other statuses can be hidden. In the case when you click on info-icon, you can show all statuses.

File: src/app/shell/workshop-details/workshop-page/reviews/reviews.component.ts [hunk 3]
```diff
@@ -68,7 +68,7 @@ export class ReviewsComponent implements OnInit, OnDestroy {
         filter((applications: Application[]) => applications?.length > 0),
         takeUntil(this.destroy$)
       ).subscribe((applications: Application[]) =>
-        this.approvedApplications = applications.filter((application: Application) => application.status = ApplicationStatus.approved))
+        this.approvedApplications = applications.filter((application: Application) => application.status = ApplicationStatus[3]))
```
Comment (dmitryy90) [inline src/app/shell/workshop-details/workshop-page/reviews/reviews.component.ts:71]: magic number

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/enum/applications.ts [hunk 4]
```diff
@@ -13,4 +13,13 @@ export enum ApplicationTitles {
   'Зараховано',
   'Відмовлено',
   'Гурток залишено',
+  'Заблоковано'
+}
+export enum ApplicationStatusDescription {
```
Comment (dmitryy90) [inline src/app/shared/enum/applications.ts:18]: please move dictionary enums to separate folder

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shell/personal-cabinet/personal-cabinet.component.ts [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewChild } from '@angular/core';
 import { Store } from '@ngxs/store';
 import { Role, RoleLinks } from 'src/app/shared/enum/role';
 import { NavBarName } from 'src/app/shared/enum/navigation-bar';
 import { User } from 'src/app/shared/models/user.model';
 import { AddNavPath, DeleteNavPath } from 'src/app/shared/store/navigation.actions';
 import { RegistrationState } from 'src/app/shared/store/registration.state';
+import { MatMenuTrigger } from '@angular/material/menu';
+// import { MatMenuTrigger } from '@angular/material/menu';
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/personal-cabinet.component.ts:10]: do we need this?

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.scss [hunk 6]
```diff
@@ -96,4 +96,15 @@
...
+// .status-menu, .application-popup-status, .mat-application-menu, .application-info-menu{
+//   width: max-content;
+//   max-width: max-content;
+//   font-size: 13px;
+//   font: Open Sans;
+//   font-weight: normal;
+// }
+// .application-info-description{
+// color: #AAAAAA;
+// }
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.scss:110]: remove please styling that is not used

File: src/app/shell/personal-cabinet/applications/applications.component.html [hunk 7]
```diff
@@ -2,36 +2,38 @@
 
 
 <ng-template #isProvider>
+  <div fxLayout="row" fxLayoutAlign="end center">
+ <app-status-info ></app-status-info>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/applications/applications.component.html:6]: add pointer of hover

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**